imagefiles/install-crosstool-ng-toolchain.sh

```diff
@@ -57,12 +67,17 @@ mkdir -p "${CTNG}"
 cd "${CTNG}"
 
 # Download and install the "crosstool-ng" source.
-REV=1.23.0
-curl -# -LO \
-  "https://github.com/crosstool-ng/crosstool-ng/archive/crosstool-ng-${REV}.tar.gz"
-tar -xf "crosstool-ng-${REV}.tar.gz"
-patch crosstool-ng-crosstool-ng-1.23.0/scripts/build/companion_libs/210-expat.sh -i /dockcross/crosstool-ng-expat.patch
-cd "crosstool-ng-crosstool-ng-${REV}"
+
+git clone https://github.com/crosstool-ng/crosstool-ng.git
+cd crosstool-ng
+git fetch --tags
+
+# checkout 
+git checkout ${REV}
```

Comment:
Why is that needed instead of using the older way with curl?

Reply:
Because the last version is more than 2 years old (https://github.com/crosstool-ng/crosstool-ng/releases), I would like to reference a commit while version 1.25 is released : https://github.com/crosstool-ng/crosstool-ng/issues/1477